<launch>

	<node name="control_system" pkg="FleLeSy" type="control_system.py" output="screen"/>
	<node name="start_modules" pkg="FleLeSy" type="start_modules.py" output="screen"/>
<!--name="$(anon control_system)"-->
<!--ns="control_system"-->
</launch>
